src/seismometer/_api.py

```diff
@@ -307,29 +395,27 @@ def _plot_cohort_hist(
     HTML
         A stacked set of histograms for each selected subgroup in the cohort.
     """
-
     cData = get_cohort_data(dataframe, cohort_col, proba=output, true=target, splits=subgroups)
 
-    # filter by group size
+    # filter by groups by size
     cCount = cData["cohort"].value_counts()
     good_groups = cCount.loc[cCount > censor_threshold].index
     cData = cData.loc[cData["cohort"].isin(good_groups)]
 
     if len(cData.index) == 0:
-        logger.error(
-            "No groups were left uncensored; timeframe is too small in combination with frequency of "
-            + f"{target_event} and the number of cohorts"
-        )
-        return
+        return template.render_censored_plot_message(censor_threshold)
 
     bins = np.histogram_bin_edges(cData["pred"], bins=20)
```

Comment:
What's special about magic number 20? Should this be a parameter in the function or a constant defined somewhere?

Reply:
20 is just a nice number of bins